Makefile

```diff
@@ -6,14 +5,25 @@ NODE_FLAGS ?= $(shell $(NODE) --v8-options | grep generators | cut -d ' ' -f 3)
 MOCHA     = $(NODE) $(NODE_FLAGS) ./node_modules/.bin/_mocha
 ISTANBUL  = $(NODE) $(NODE_FLAGS) ./node_modules/.bin/istanbul
 COVERALLS = $(NODE) $(NODE_FLAGS) ./node_modules/.bin/coveralls
+BABEL     = $(NODE) $(NODE_FLAGS) ./node_modules/.bin/babel
 
 # Install dependencies with npm.
 node_modules: package.json
 	@npm install
 	@touch node_modules # hack to fix mtime after npm installs
 
+# Build
+build: build-src build-bin
+build-src: src
+	@$(BABEL) src -d lib
+build-bin: bin-src
+	@-mkdir bin
+	$(BABEL) bin-src/metalsmith > bin/metalsmith
+	$(BABEL) bin-src/_metalsmith > bin/_metalsmith
+	@chmod u+x bin/*
```

Comment:
You might be able to tweak this to be a bit more elegant and more efficient on repeated builds. make can be a surprisingly powerful tool in situations like this. Definitely not a big deal, but I'll look into it later today.